flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/DatabaseSync.java

```diff
@@ -108,15 +116,23 @@ public void build() throws Exception {
         List<String> dorisTables = new ArrayList<>();
         for (SourceSchema schema : schemaList) {
             syncTables.add(schema.getTableName());
-            String dorisTable = converter.convert(schema.getTableName());
+            // 转换之前得判断是否是多表合一,多表合一不进行转换
+            String dorisTable="";
+            if(converter.isMultiToOneTable(schema.getTableName())){
+                dorisTable = converter.getMultiToOneTargetByTableName(schema.getTableName());
+            }else{
+                dorisTable = converter.convert(schema.getTableName());
+            }
```

Comment:
Can it be included in a method and handle the specific logic in the convert method?